tests/integration/netlify-forms.test.ts

```diff
@@ -0,0 +1,31 @@
+import { getLogger } from 'lambda-local'
+import { v4 } from 'uuid'
+import { beforeEach, expect, it, vi } from 'vitest'
+import { type FixtureTestContext } from '../utils/contexts.js'
+import { createFixture, runPlugin } from '../utils/fixture.js'
+import { generateRandomObjectID, startMockBlobStore } from '../utils/helpers.js'
+
+getLogger().level = 'alert'
+
+beforeEach<FixtureTestContext>(async (ctx) => {
+  // set for each test a new deployID and siteID
+  ctx.deployID = generateRandomObjectID()
+  ctx.siteID = v4()
+  vi.stubEnv('SITE_ID', ctx.siteID)
+  vi.stubEnv('DEPLOY_ID', ctx.deployID)
+  vi.stubEnv('NETLIFY_PURGE_API_TOKEN', 'fake-token')
+  // hide debug logs in tests
+  // vi.spyOn(console, 'debug').mockImplementation(() => {})
+
+  await startMockBlobStore(ctx)
+})
+
+it.skip<FixtureTestContext>('test', async (ctx) => {
```

Comment:
is it worth checking if `console.warn` was called? this test (and fixture) right now doesn't do anything (I guess until warning is swapped to build failing)?

Reply:
i wondered this, but since the dry run is a temporary solution i didn't think it was worth it

Reply:
Also fine with pushing this in - but maybe we could name the test already?
```suggestion
// test skipped until we actually start failing builds - right now we are just showing a warning
it.skip<FixtureTestContext>('should fail build when netlify forms are used', async (ctx) => {
```